localisation/00_wc_sad.csv

```diff
@@ -749,7 +749,21 @@ RULE_STATIC_PORTRAITS_ENABLED;Enabled;;Aktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_ENABLED_DESC;Static portraits for certain lore characters are �Genabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Gaktiviert�!.;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED;Disabled;;Deaktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED_DESC;Static portraits for certain lore characters are �Rdisabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Rdeaktiviert�!.;;;;;;;;;;;x
-;;;;;;;;;;;;;;
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+writing_whispers;Hears Whispers;;;;;;;;;;;;;x
+writing_whispers_desc;This character hears the whispers of the Old Gods.;;;;;;;;;;;;;x
+whispers_tooltip_tip;Quality of the book depends on amount of your �YTrueReligion.GetPietyName�!.;;;;;;;;;;;;;x
```

Comment:
Continuation of

> Personally, I would make a new line using \n in hermetics_write_magnum_opus_desc instead of custom_tooltip in allow block, so player can see it more easily.

```suggestion

```